poetry/masonry/utils/tags.py

```diff
@@ -75,11 +75,16 @@ def get_abi_tag(env):
         if get_flag(
             env,
             "Py_DEBUG",
-            lambda: hasattr(sys, "gettotalrefcount"),
-            warn=(impl == "cp"),
+            lambda: (hasattr(sys, "gettotalrefcount") and env.version_info < (3, 8)),
+            warn=(impl == "cp" and env.version_info < (3, 8)),
```

Comment:
debug flag "u" doesn't require python < 3.8

Reply:
I get the following runtime warning with Python 3.8:
```
RuntimeWarning: Config variable 'Py_DEBUG' is unset, Python ABI tag may be incorrect
```
So, at least there should be no warning if the flag is not present.

Reply:
It makes sense